music21/converter/__init__.py

```diff
@@ -632,11 +632,16 @@ def parseURL(self, url, format=None, number=None, **keywords):  # @ReservedAssig
         Note that this checks the user Environment
         `autoDownload` setting before downloading.
 
+        >>> us = environment.UserSettings()
+        >>> us['autoDownload'] = 'allow'
```

Comment:
We shouldn't be changing UserSettings during any test scripts.  in particular here, we're going to wipe out whatever the user's autoDownload already was and set it to deny afterwards.  I think that this should be set to what it was before and we can use environment.Environment() in a unit test instead.